core/lib/server/form.ts

```diff
@@ -183,3 +194,100 @@ export const createFormInviteLink = async (props: FormInviteLinkProps) => {
 
 	return magicLink;
 };
+
+const componentsBySchemaTable = Object.entries(componentsBySchema)
+	.map(([schema, components]) => {
+		const component = components[0]
+			? `'${components[0]}'::"InputComponent"`
+			: `null::"InputComponent"`;
+		return `('${schema}'::"CoreSchemaType", ${component})`;
+	})
+	.join(", ");
+
+export const insertForm = (
```

Comment:
I went with `insert` here because I was extracting the query portion of the `createForm` action and I wanted to distinguish between the server action function and the database query function. My thinking was that `insert` is the name of the database operation, while `create` is the action a user wants to take. I'm open to something else but I was also following a pattern I used recently when trying to avoid having to alias the query functions when we have to import them in a server action like [we used to do on the members page](https://github.com/pubpub/platform/blob/c118f6b9a98291001debf2ac026502943c7b61ee/core/app/c/%5BcommunitySlug%5D/members/%5B%5B...add%5D%5D/actions.ts#L16-L20) before I made a similar change.